### DISCUSSION TITLE
How to Remove Empty Line At The Top in frontcube theme

### DISCUSSION THREAD

**Discussion by isknurul007** [ohmyzsh/ohmyzsh#10988] [Help] [answered]
Hi,
I am using **frontcube** theme, how do I remove the empty line at the top ?

<img width="454" alt="frontcube" src="https://user-images.githubusercontent.com/106715990/173217281-e24ea88e-ac60-436d-a452-602d00507589.png">

**Comment by mcornella** [comment, marked as answer]
The `frontcube` theme adds an empty line to the PROMPT by design. You can follow [this guide](https://github.com/ohmyzsh/ohmyzsh/wiki/Customization#overriding-and-adding-themes) to copy the theme to the [`$ZSH_CUSTOM` folder](https://github.com/ohmyzsh/ohmyzsh/wiki/Settings#zsh_custom) and remove the newline from the prompt, so https://github.com/ohmyzsh/ohmyzsh/blob/d41ca84af1271e8bfbe26f581cebe3b86521d0db/themes/frontcube.zsh-theme#L2-L4
would look like
```sh
PROMPT='$fg_bold[gray]%}%~%{$fg_bold[blue]%}%{$fg_bold[blue]%} % %{$reset_color%} 
%{$fg[green]%}➞  %{$reset_color%'
```
